homeassistant/components/matter/__init__.py

```diff
@@ -82,6 +121,25 @@ async def async_remove_entry(hass: HomeAssistant, entry: ConfigEntry) -> None:
     storage_path = get_matter_store(hass, entry).path
     await hass.async_add_executor_job(Path(storage_path).unlink)
 
+    if not entry.data.get(CONF_INTEGRATION_CREATED_ADDON):
+        return
+
+    addon_manager: AddonManager = get_addon_manager(hass)
+    try:
+        await addon_manager.async_stop_addon()
+    except AddonError as err:
+        LOGGER.error(err)
```

Comment:
```suggestion
        LOGGER.error("Error stopping add-on: %s", err)
```

Reply:
The addon manager already has an error message in the exception.

https://github.com/home-assistant/core/blob/effa9940ff185289dbad9ab8c88b72f7067184f5/homeassistant/components/hassio/addon_manager.py#L58-L60